torch/_dynamo/variables/tensor.py

```diff
@@ -707,15 +710,25 @@ def var_getattr(self, tx, name):
             result = wrap_fx_proxy_cls(
                 target_cls=NumpyNdarrayVariable,
                 tx=tx,
-                proxy=GetAttrVariable.create_getattr_proxy(self.as_proxy(), name),
+                proxy=tx.output.create_proxy(
+                    "call_function",
+                    attr_wrapper,
+                    (self.as_proxy(), name),
+                    {},
+                ),
                 example_value=None,
                 **options,
             )
         elif name in ["size", "itemsize", "strides", "shape", "ndim"]:
             result = wrap_fx_proxy_cls(
                 target_cls=ConstantVariable,
                 tx=tx,
-                proxy=GetAttrVariable.create_getattr_proxy(self.as_proxy(), name),
+                proxy=tx.output.create_proxy(
+                    "call_function",
+                    attr_wrapper,
+                    (self.as_proxy(), name),
+                    {},
+                ),
```

Comment:
I'm not entirely sure about this, but I'm not done reviewing. What I'm hoping to see here is that the logic for handling these looks substantively similar to how TensorVariable; I would not necessarily try to force it through torch_np.